### PR TITLE
naukri-link-added

### LINKED ISSUE #2271: [Add] Naukri
### Site URL

https://www.naukri.com/

### Description of the above provided site

It's a platform that helps you get the job and place the job

### Category

Other

### Sub Category

Cat - Placement_prep , Subcat - jobportals

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by parma1111**
## Fixes Issue

 Closes #2271 

## Changes proposed

It's a platform that helps you get the job and place the job

## Screenshots

Screenshot (901)
### updated-description
Screenshot (902)

### CODE REVIEW

File: database/placement_prep/job-portals.json [hunk 1]
```diff
@@ -47,5 +47,12 @@
     "url": "https://monster.com",
     "category": "placement-prep",
     "subcategory": "job-portals"
+  },
+  {
+    "name": "Naukri",
+    "description": "It's a platform that helps you get the job and place the job.",
```
Comment (CBID2) [inline database/placement_prep/job-portals.json:53]: [proofreading]: there was a part of the description that was unclear. Consider describing the source in the suggestion below
```suggestion
    "description": "It's a platform that helps you find and get the job.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @parma1111. Your PR is off to a good start. The description of your chosen resource could use some work, so I provided some advice on how to improve it. :)

**Comment by parma1111** [comment]
> Hi @parma1111. Your PR is off to a good start. The description of your chosen resource could use some work, so I provided some advice on how to improve it. :)

> Now review the PR @CBID2

**Review by CBID2 — APPROVED** [review]
Looks good to me! :) You're turn @Anmol-Baranwal

**Comment by Anmol-Baranwal** [comment]
@rupali-codes 
Deployment!

**Review by rupali-codes — APPROVED** [review]
lgtm

**Review by aftabrehan — COMMENTED** [review]
I can't access this website.

Screenshot 2024-03-03 at 11 52 07 AM

**Comment by CBID2** [comment]
> I can't access this website.
> 
> 
> 
> Screenshot 2024-03-03 at 11 52 07 AM
> 
> 

It works for me @aftabrehan 
screenshot of naurki.com

**Comment by aftabrehan** [comment]
I guess it's an issue from my side. Feel free to merge it then.

**Review by aftabrehan — APPROVED** [review]

**Comment by CBID2** [comment]
@rupali-codes, we got a merge issue here

**Comment by rupali-codes** [comment]
> @rupali-codes, we got a merge issue here

what's it

**Comment by CBID2** [comment]
> > @rupali-codes, we got a merge issue here
> 
> 
> 
> what's it

For some odd reason, we can't merge this Pull Request despite approving it.

**Comment by rupali-codes** [comment]
> > > @rupali-codes, we got a merge issue here
> > 
> > 
> > what's it
> 
> For some odd reason, we can't merge this Pull Request despite approving it.

what is it showing? screenshot please

**Comment by rupali-codes** [comment]
> > > > > @rupali-codes, we got a merge issue here
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > what's it
> > 
> > 
> > > 
> > 
> > 
> > > For some odd reason, we can't merge this Pull Request despite approving it.
> > 
> > 
> > what is it showing? screenshot please
> 
> image

Well, i don't know why is it showing this, but let me merge it for now

**Pull request merged**